modules/@apostrophecms/schema/ui/apos/logic/AposInputRelationship.js

```diff
@@ -127,11 +127,11 @@ export default {
     validate(value) {
       this.checkLimit();
 
-      if (this.field.required && !value.length) {
+      if (this.field.required && !value?.length) {
         return { message: 'required' };
       }
 
-      if (this.field.min && this.field.min > value.length) {
+      if (this.field.min && this.field.min > value?.length) {
```

Comment:
12 > null is considered `true`... In this case it works but I feel bad about using this js weirdness 😅

Reply:
I think it's rather helpful in this case 😄